common.mk

```
@@ -7,15 +7,23 @@ relpath = $(dir $(lastword $(MAKEFILE_LIST)))
 # Default values
 distname ?= $(notdir ${CURDIR})
 releases ?= $(relpath)releases/${distname}
+versions ?= versions
+
+nightlyVer = nightly-$(shell date '+%Y-%m-%d')
```

In case Actions use worker with different timezone, lets ensure UTC
```suggestion
nightlyVer = nightly-$(shell date -u '+%Y-%m-%d')
```